src/copier/copy-hitsounds.ts

```diff
@@ -0,0 +1,72 @@
+import type { Options } from "./types";
+
+import { Circle, Slider, Spinner, StandardBeatmap } from "osu-standard-stable";
+import { convertBeatmapToHitsoundableTimeLine } from "./utils";
+import {
+  copyCircleHitsounds,
+  copySliderHitsounds,
+  copySpinnerHitsounds,
+} from "./hitsounds/service";
+import { copyVolumes } from "./volumes/services/copy";
+
+export const copyHitsounds = (
+  originBeatmap: StandardBeatmap,
+  destinationBeatmap: StandardBeatmap,
+  options: Options
+): StandardBeatmap => {
+  const hitsoundedBeatmap = structuredClone(destinationBeatmap);
```

Comment:
This is actually causing problems during the beatmap encoding, i think we should keep it as just a reference.